src/components/Geolocation/Geolocation.js

```diff
@@ -60,8 +53,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  className: 'tm-geolocation',
-  classNameActive: 'blink',
+  className: 'rs-geolocation',
   title: undefined,
```

Comment:
title and className can be remove using spread props int eh render method.

Reply:
To support the `active` class I kept the className prop. Also this improves documentation so we should think about doing this always.

Reply:
MAybe  you 're rigth. Can you add the classNme  for all the other components? thx